bin/versions.sh

```diff
@@ -24,7 +24,7 @@ fi
 
 if [[ $LANGUAGES == *":java:"* ]]; then
     echo -n java,
-    java -version 2>&1 |grep "version" | cut -f3 -d " " | cut -c 2-9
+    java -version 2>&1 |grep "version" | cut -f2 -d "\""
```

Comment:
While you're at it, would you mind also adding a space between the pipe and `grep`? The inconsistency bothers me a bit :sweat_smile:

Reply:
haha sure!

Reply:
For some reason the github PR is not updating from the merging branch. Maybe it just needs some time to update

Reply:
ended up having to fp